docs/source/en/api/pipelines/text_to_video_zero.md

```diff
@@ -35,25 +30,25 @@ Our key modifications include (i) enriching the latent codes of the generated fr
 Experiments show that this leads to low overhead, yet high-quality and remarkably consistent video generation. Moreover, our approach is not limited to text-to-video synthesis but is also applicable to other tasks such as conditional and content-specialized video generation, and Video Instruct-Pix2Pix, i.e., instruction-guided video editing.
 As experiments show, our method performs comparably or sometimes better than recent approaches, despite not being trained on additional video data.*
 
-You can find additional information about Text-to-Video Zero on the [project page](https://text2video-zero.github.io/), [paper](https://arxiv.org/abs/2303.13439), and [original codebase](https://github.com/Picsart-AI-Research/Text2Video-Zero).
+You can find additional information about Text2Video-Zero on the [project page](https://text2video-zero.github.io/), [paper](https://arxiv.org/abs/2303.13439), and [original codebase](https://github.com/Picsart-AI-Research/Text2Video-Zero).
 
 ## Usage example
 
 ### Text-To-Video
 
-To generate a video from prompt, run the following python command
+To generate a video from prompt, run the following python code:
 ```python
 import torch
-import imageio
 from diffusers import TextToVideoZeroPipeline
+from diffusers.utils import export_to_video
 
 model_id = "runwayml/stable-diffusion-v1-5"
 pipe = TextToVideoZeroPipeline.from_pretrained(model_id, torch_dtype=torch.float16).to("cuda")
 
 prompt = "A panda is playing guitar on times square"
 result = pipe(prompt=prompt).images
 result = [(r * 255).astype("uint8") for r in result]
-imageio.mimsave("video.mp4", result, fps=4)
+export_to_video(result, "video.mp4")
```

Comment:
I'm unable to reproduce your error here, and the generated video isn't meaningful (just a green screen?)

Reply:
I'm using Kaggle, and it throws this for `imageio.mimsave("video.mp4", result, fps=4)` -> TypeError: TiffWriter.write() got an unexpected keyword argument 'fps'.
And this is the result of `export_to_video(result, "video.mp4")`.

https://github.com/huggingface/diffusers/assets/46008593/c14f1e49-0afa-434e-aa4e-286f5359aef7

Reply:
Let's keep the current code since it's not actually broken, and maybe you can open an Issue and report this error with `imageio.mimsave`. Also still not seeing anything in the video you uploaded

Reply:
Download it, then watch.

Reply:
Yes, Colab, with its version of imageio, can run the code. Colab's imageio version is newer than Kaggle's, so we can wait for Kaggle to upgrade. I am reversing these changes.

Reply:
Sure, feel free to try and adapt the `export_to_video` function for the `TextToVideoPipelineOutput`! Otherwise, it'd also be ok to just reverse the changes

Reply:
Reversed.